src/diffusers/loaders/single_file_utils.py

```diff
@@ -92,6 +97,7 @@
         "double_blocks.0.img_attn.norm.key_norm.scale",
         "model.diffusion_model.double_blocks.0.img_attn.norm.key_norm.scale",
     ],
+    "autoencoder_dc": "decoder.stages.0.op_list.0.main.conv.conv.weight",
```

Comment:
We would need to infer the model repo type using this key right? That still has to be added.

Reply:
Oh sorry, missed it. Adding now, but not sure how this worked before then 🤔